test/IntegrationTests/SmokeTests.cs

```diff
@@ -218,6 +218,47 @@ public void ZipkinExporter()
         }
     }
 
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void ZipkinAndOtlpTracesExporter()
```

Comment:
@open-telemetry/dotnet-instrumentation-approvers, what do you think `ZipkinTracesExporter` and `OtlpTracesExporter` to reduce the execution of integration tests? Basically this test combines both tests.

Reply:
Hum I like an integration test that we create multiple exporters and check their outputs.

Reply:
I mean. Keep this test and remove the "old ones" :D

Reply:
Let's keep as it is, i can remove in separate PR